docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -1268,180 +1268,193 @@ The provided replicas <1>, labels <2> and environment variables <3>  were retain
 
-=== Generating ServiceBinding resources
-The <<service_binding, Service Binding>> covers, how the application will consume the bound service. 
-However, we still haven't touched on how the actual binding happens. The binding is the responsibility of the Service Binding Operator. The user provides a `ServiceBinding` resource which expresses which application will bound to which services and the operator is responsible for mounting the required files to container.
+The Service Binding Operator, responsible for the binding process, then reads the information from the `ServiceBinding` resource and mounts the required files to a container accordingly.
 
-A `ServiceBinding` looks like:
 
+* An example of the `ServiceBinding` resource:
++
 [source,yaml]
 ----
 apiVersion: binding.operators.coreos.com/v1alpha1
 kind: ServiceBinding
 metadata:
-  name: binding-request
-  namespace: service-binding-demo
+ name: binding-request
+ namespace: service-binding-demo
 spec:
-  application:
-    name: java-app
-    group: apps
-    version: v1
-    resource: deployments
-  services:
-  - group: postgresql.baiju.dev
-    version: v1alpha1
-    kind: Database
-    name: db-demo
-    id: postgresDB
+ application:
+   name: java-app
+   group: apps
+   version: v1
+   resource: deployments
+ services:
+ - group: postgresql.baiju.dev
+   version: v1alpha1
+   kind: Database
+   name: db-demo
+   id: postgresDB
 ----
-
-The `quarkus-kubernetes-service-binding` extension provides a more compact way of expressing this information:
++
+[NOTE]
+====
+The `quarkus-kubernetes-service-binding` extension provides a more compact way of expressing the same information:
 
 [source,properties]
 ----
 quarkus.kubernetes-service-binding.services.db-demo.api-version=postgresql.baiju.dev/v1alpha1
 quarkus.kubernetes-service-binding.services.db-demo.kind=Database
 ----
+====
 
-By adding the above configuration properties inside your `application.properties` the `quarkus-kubernetes` in cooperation with the `quarkus-kubernetes-service-binding` extension will generate the `ServiceBinding` for you.
-And this is something that is meant to work regardles of the kubernets flavour you are using, so it's expected to also work for Openshift and Knative, without any additional action from the user.
+After adding the above configuration properties inside your `application.properties`, the `quarkus-kubernetes`, in combination with the `quarkus-kubernetes-service-binding` extension, will automatically generate the `ServiceBinding` resource.
 
-In the configuration above `db-demo` has a double role. It is used to correlate / group `api-version` and `kind` together, but it's also used as the `name` of the custom resource. The later can be overriden if needed. For example:
+Afterwards, the above configuration `db-demo` yaml file:
```

Comment:
I think that this is a bit off. `db-demo` is not a yaml file.
It is an identifier that is used in the properties configuration above, that is used to identify the service. Also since in the configuration no name is explicitly specified this indentifier is also used as a name.

To be honest I feel that the original description is more accurate.

Reply:
What about this?

The above mentioned `db-demo` property-configuration identifier has then a double role in which:

* correlates and groups `api-version` and `kind` together
* defines `name` of the custom resource with a possibility for a later edit:


Applying your suggestions while provides better readability :)